config_example.sh

```diff
@@ -496,6 +486,26 @@ set -x
 #export MASTER_DISK=40
 #export MASTER_VCPU=8
 
+# ARBITER_HOSTNAME_FORMAT -
+# Set a custom hostname format for arbiters. This is a format string that should
+# include one %d field, which will be replaced with the number of the node.
+# Default: "arbiter-%d"
+#
+#export ARBITER_HOSTNAME_FORMAT=arbiter-%d
+
+# NUM_ARBITERS -
+# Indicate number of arbiter nodes to deploy.
+# Arbiter nodes take the place of a master node but run only critical containers to maintain HA for the cluster,
+# it is a TechPreview for OCP 4.19, so `export FEATURE_SET="TechPreviewNoUpgrade"` must be set.
```

Comment:
@eggfoobar can you please add the proper doc for the FEATURE_SET new config var as well (ok also in a subsequent PR) ? As per the current implementation it could be set indipendently (and probably in future also another var for the feature gates could be useful)

Reply:
Sure thing, it was a pre-existing flag that I saw in the code and didn't notice there was no doc for it. I'll open a PR to add that and some feature gate addons would be nice as well. Thanks!